tests/cypress/integration/actions_tasks_objects/case_18_filters_functionality.js

```diff
@@ -103,46 +104,83 @@ context('Filters functionality.', () => {
+            const textFilter = 'attr["count points"] == "4"';
+            cvatFiltesList.push(textFilter);
+            cy.writeFilterValue(true, textFilter); // #cvat_canvas_shape_2,4, #cvat-objects-sidebar-state-item-2,4
             checkingFilterApplication([2, 4]);
         });
         it('Filter: width >= height. All objects exist.', () => {
-            cy.writeFilterValue(true, 'width >= height'); // #cvat_canvas_shape_1,2,3,4, #cvat-objects-sidebar-state-item-1,2,3,4
+            const textFilter = 'width >= height';
+            cvatFiltesList.push(textFilter);
+            cy.writeFilterValue(true, textFilter); // #cvat_canvas_shape_1,2,3,4, #cvat-objects-sidebar-state-item-1,2,3,4
             checkingFilterApplication([1, 2, 3, 4]);
         });
         it('Filter: clientID == 4. Only the objects with same id exist (polygon track).', () => {
-            cy.writeFilterValue(true, 'clientID == 4'); // #cvat_canvas_shape_7, #cvat-objects-sidebar-state-item-4
+            const textFilter = 'clientID == 4';
+            cvatFiltesList.push(textFilter);
+            cy.writeFilterValue(true, textFilter); // #cvat_canvas_shape_7, #cvat-objects-sidebar-state-item-4
             checkingFilterApplication([4]);
         });
         it('Filter: (label=="shape 3 points" & attr["polylines"]==true) | (label=="track 4 points" & width > 60). Only the objects polygon and rectangle exist.', () => {
-            cy.writeFilterValue(
-                true,
-                '(label=="shape 3 points" & attr["polylines"]==true) | (label=="track 4 points" & width > 60)',
-            ); // #cvat_canvas_shape_2,4, #cvat-objects-sidebar-state-item-2,4
+            const textFilter =
+                '(label=="shape 3 points" & attr["polylines"]==true) | (label=="track 4 points" & width > 60)';
+            cvatFiltesList.push(textFilter);
+            cy.writeFilterValue(true, textFilter); // #cvat_canvas_shape_2,4, #cvat-objects-sidebar-state-item-2,4
             checkingFilterApplication([2, 4]);
         });
         it('Filter: (( label==["shape 3 points"]) | (attr["type"]=="shape" & width > 50)) & (height > 50 & (clientID == serverID))). All objects not exist.', () => {
-            cy.writeFilterValue(
-                true,
-                '(( label==["points shape"]) | (attr["type"]=="shape" & width > 50)) & (height > 50 & (clientID == serverID)))',
-            );
+            const textFilter =
+                '(( label==["points shape"]) | (attr["type"]=="shape" & width > 50)) & (height > 50 & (clientID == serverID)))';
+            cvatFiltesList.push(textFilter);
+            cy.writeFilterValue(true, textFilter);
             checkingFilterApplication([]);
         });
+        it('Verify to show all filters', () => {
+            cvatFiltesList.forEach(function (filterValue) {
+                cy.contains('.cvat-annotations-filters-input-history-element', filterValue)
```

Comment:
``cy.contains`` should be enough. Isn't it?

Reply:
Yes, sure. Unnecessary actions removed.